src/network/mod.rs

```diff
@@ -22,16 +22,22 @@ use std::fmt;
 use std::io;
 use std::error;
 
-pub mod constants;
-
 pub mod address;
-pub use self::address::Address;
+pub mod constants;
 pub mod message;
 pub mod message_blockdata;
 pub mod message_network;
 pub mod message_filter;
 pub mod stream_reader;
 
+// Re-export all network types.
+pub use self::address::*;
+pub use self::constants::*;
+pub use self::message::*;
+pub use self::message_blockdata::*;
+pub use self::message_network::*;
+pub use self::message_filter::*;
```

Comment:
Can you make an explicit list?

It will be long but using `*` makes the codebase ungreppable.

Reply:
I'm concerned about types being forgotten when explicit listing. How do you(/we) feel about making the submodules private so that all types have to be exported (otherwise cargo will shout unused).